app/routes/cdo/search/basic.js

```diff
@@ -0,0 +1,45 @@
+const { routes, views } = require('../../../constants/search')
+const { getSearchCriteria, setSearchCriteria, getSearchResults, setSearchResults } = require('../../../session/cdo/search')
+const ViewModel = require('../../../models/cdo/search/basic')
+const searchSchema = require('../../../schema/portal/search/basic')
+const { doSearch } = require('../../../api/ddi-index-api/search')
+const { admin } = require('../../../auth/permissions')
+
+module.exports = [{
+  method: 'GET',
```

Comment:
It's a GET when passing the request to the API. However, I thought it was cleaner to do a POST (in separation to the GET to display the form itself) since we're posting data to the backend (and validating) as opposed to a simple call with parameters.
Happy to change if you think it makes it easier for people to maintain it

Reply:
I think it should be a GET as we're not actually changing anything on the API side. We're just requesting data here. Making it a GET and having the parameters in the query string would also solve the above comment.